src/large_scale_condensation.jl

```diff
@@ -0,0 +1,43 @@
+function get_large_scale_condensation_tendencies!(
+    Diag::DiagnosticVariables{NF},
+    M   ::ModelSetup,
+) where {NF<:AbstractFloat}
+    @unpack gravity, RH_thresh_max, RH_thresh_range, RH_thresh_boundary, humid_relax_time = M.constants
+    @unpack cp, alhc = M.parameters
+    @unpack nlon, nlat, nlev, σ_levels_full, σ_levels_thick = M.geospectral.geometry
+    @unpack temp_grid, humid_grid, pres_surf_grid = Diag.grid_variables
+    @unpack temp_tend, humid_tend, sat_spec_humidity, sat_vap_pressure, cloud_top, precip_large_scale = Diag.parametrization_variables
+
+    pres = exp.(pres_surf_grid)  # Normalised surface pressure - TODO(alistair): check pressure units throughout
```

Comment:
ideally we preallocate `pres` somewhere else and reuse it here. As pressure is always surface pressure maybe we should also call it `pres_log_grid` and `pres_grid` to better distinguish between the logarithm of surface pressure and non-log version?

Reply:
I would be in favour of renaming `pres_surf_grid` to `pres_log_grid`, because I found it hard to figure out what to call `exp.(pres_surf_grid)` :sweat_smile: 

Question about units: is pressure always hPa?

Reply:
Also, if we preallocate `pres_grid` somewhere else, then we have to make sure to update it everytime `pres_log_grid` changes. Where exactly should we do that? So we don't accidentally take the exponential twice, or something.